eZ/Publish/API/Repository/Repository.php

```diff
@@ -19,6 +19,8 @@
 interface Repository
 {
     /**
+     * @deprecated since 6.5, to be removed. Use PermissionResolver::getCurrentUserReference() instead.
```

Comment:
I might be wrong here, but I think we should deprecate since `6.6` because `6.5` has been released already.